test/integration/targets/iosxr_system/tests/netconf/set_hostname.yaml

```diff
@@ -8,11 +8,13 @@
       lines: hostname switch
       match: none
       provider: "{{ cli }}"
+    connection: network_cli
 
   - name: configure hostname
     iosxr_system:
       hostname: foo
       provider: "{{ netconf }}"
```

Comment:
provider is not required if the connection is set to netconf in the task

Reply:
Same as above!